crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -1390,10 +1390,15 @@ pub enum ServerDriveIoRequest {
     ServerDriveQueryInformationRequest(ServerDriveQueryInformationRequest),
     DeviceCloseRequest(DeviceCloseRequest),
     ServerDriveQueryDirectoryRequest(ServerDriveQueryDirectoryRequest),
+    /// A special type signifying that the minor function is not supported.
+    /// This is not a real PDU type, rather it is used to pass control of
+    /// what to do with the unsupported minor function to the caller.
+    UnsupportedMinorFunctionRequest(DeviceIoRequest),
```

Comment:
Issue (non-blocking): it seems to suffer from the ["fallback variant problem"](https://github.com/Devolutions/IronRDP/blob/b554c6a6d1ecc7f7070b5fcebc3d78fd2bdbfbbc/crates/ironrdp-pdu/README.md#enumeration-like-types-should-allow-resilient-parsing), and is also not really part of `DR_DRIVE_CORE_DEVICE_IOREQUEST` so it’s not really ideal. Fixing this is probably not very straightforward however, I’ll not block the PR for that.